.pipelines/osm-arc-prod.yaml

```diff
@@ -0,0 +1,19 @@
+jobs: 
+  - job: test_osm_arc_prod
+    pool: staging-pool
+    workspace: 
+      clean: all
+    steps:
+    - template: templates/aks-setup.yaml
+    - template: templates/enable-arc.yaml
+    - template: templates/add-arc-extension.yaml
+      parameters: 
+        imageTag: $(EXTENSION_VERSION)
+        releaseTrain: $(EXTENSION_RELEASE_TRAIN)
```

Comment:
what's the expected release train for prod?

Reply:
it could be either pilot, preview, or stable. Until GA we will be using pilot, once we reach GA then either preview or stable.